### PR TITLE
feat: I have come an end with my CV in pug

### LINKED ISSUE #22: add a header with the title
“where you can contact me” plus  the youtube icon underneath

### LINKED ISSUE #23: add the  contact section
with the information needed

### LINKED ISSUE #32: add the buttons that will include all the socials where you can be contacted

### PR DESCRIPTION
**Pull request by Alez05**
fix #23 fix #22 fix #32

### CODE REVIEW

**Review by Vivy — APPROVED** [review]
good

**Review by flivanov92 — COMMENTED** [review]
👍

**Review by stefantanasa — APPROVED** [review]
Nice! Try mixins, they're great. Give it a try on the next pug project

**Review by florin-bizgan — APPROVED** [review]
̿̿ ̿̿ ̿̿ ̿'̿'\̵͇̿̿\з= ( ▀ ͜͞ʖ▀) =ε/̵͇̿̿/’̿’̿ ̿ ̿̿ ̿̿ ̿̿

File: src/_component/contact/contact.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+.box-body {
+  // position: absolute;
+  width: 320px;
+  position: relative;
+  flex-basis: 60%;
+  // top: 85px;
+  background-color: antiquewhite;
+  padding: 0 8px;
+  // justify-content: center;
```
Comment (MariaTeo) [inline src/_component/contact/contact.scss:44]: you could delete commented code if you don't use it

**Review by MariaTeo — APPROVED** [review]

**Pull request merged**